content/reference/drops/time_drop.md

```diff
@@ -0,0 +1,14 @@
+# Time Drop
+
+Return attributes with time. You have ability to manage time format.
```

Comment:
@Fredar this is used in couple places but see the confusion at first. However, that follows Rails convention here; and we could extend this drop with Time.now for example. Feels legit?